content/docs/0.8.1/volumes-and-nodes/create-volumes.md

```diff
@@ -2,20 +2,52 @@
 title: Create Longhorn Volumes
 weight: 1
 ---
-Before you create Kubernetes volumes, you must first create a storage class. Use following command to create a StorageClass called `longhorn`.
 
-```
-kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/master/examples/storageclass.yaml
-```
+In this tutorial, you'll learn how to create Kubernetes persistent storage resources of persistent volumes (PVs) and persistent volume claims (PVCs) that correspond to Longhorn volumes. You will use kubectl to dynamically provision storage for workloads using a Longhorn storage class. For help creating volumes from the Longhorn UI, refer to [this section.](#creating-longhorn-volumes-with-the-longhorn-ui)
 
-Now you can create a pod using Longhorn like this:
-```
-kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/master/examples/pvc.yaml
-```
+> This section assumes that you understand how Kubernetes persistent storage works. For more information, see the [Kubernetes documentation.](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
 
-The above yaml file contains two parts:
+### Creating Longhorn Volumes with kubectl
 
-1. Create a PVC using Longhorn StorageClass.
+First, you will create a Longhorn StorageClass. The Longhorn StorageClass contains the authorization to provision persistent volumes.
```

Comment:
`contains the parameters to provision...`

We don't support authorization/authentication currently.

Reply:
updated